internal/service/cloudprovideraccess/resource_cloud_provider_access_authorization.go

```diff
@@ -260,7 +260,7 @@ func resourceCloudProviderAccessAuthorizationStateUpgradeV0(ctx context.Context,
 }
```

Comment:
changes in cloud_provider_access resources are the same as in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2526/files#diff-2d8a9b4bcd0626d79ae9b41b10ef1c0ed6a1cf11ad3c0c34fc100fa0bf60f344, coming from adopting latest SDK. Will make sure solution is synced to avoid any conflicts later on.

Reply:
can we capture this in [CLOUDP-267670](https://jira.mongodb.org/browse/CLOUDP-267670) ?

Reply:
yes, added details, great that we have this ticket created already